content/sql/concepts/commands/terms/select/select.md

```diff
@@ -16,36 +16,36 @@ CatalogContent:
   - 'paths/design-databases-with-postgresql'
 ---
 
-Fetchs data from a database. Every query will begin with `SELECT`.
+Every SQL query will begin with the `SELECT` command to fetch data from one or more tables.
 
 ## Syntax
 
-The `SELECT *` statement returns all columns from the provided table in the result set:
+The `SELECT` command is followed by a list of column name to be returned from the table(s) listed in the `FROM` clause:
 
 ```sql
-SELECT *
-FROM table_name;
+SELECT column1, column2, ... columnN FROM table;
 ```
 
-You can also select individual columns:
+`SELECT` can also use the special `*` character to represent all columns from the table(s):
 
 ```sql
-SELECT column_name(s)
-FROM table_name;
+SELECT * FROM table;
 ```
 
+**Note:** It is best practice to never rely on column order when using `*`.
+
 ## Examples
 
-The following query will fetch all columns and records (rows) from the `movies` table:
+The following example selects all columns from `table1` and `table2`:
```

Comment:
Or maybe we can take an opportunity to link to the entry that includes [inner join](https://www.codecademy.com/resources/docs/sql/joins). My thought is that a reader looking on docs for `SELECT` would not know `INNER JOIN` first, so it may be good to include that here.

Maybe something like:

```suggestion
The following example selects all columns from `table1` and `table2` using an [inner join]((https://www.codecademy.com/resources/docs/sql/joins)):
```

Reply:
Should we say "INNER JOIN", though?